### DISCUSSION TITLE
ICS Calendar Time an Hour Off

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1113] [Q&A]
It's been brought to my attention the meeting ics files are generating an hour late. I was told this wasn't always the case and wonder if the time change has anything to do with this.

This is also true to with Meeting Guide app, not just through the website. I tested this out on my iPhone and iPad and it is correct, however under Windows, it is not. See images below.

![Rick](https://user-images.githubusercontent.com/62247421/230491197-0ad8e371-f677-4d75-8226-1eda958f74b4.png)

![Windows ICS](https://user-images.githubusercontent.com/62247421/230491238-26e6990e-a303-4ba8-908a-816231e57855.png)

**Comment by joshreisner** [comment]
@Grateful-Viv i think this may be a problem with the windows / outlook time zone setting - we have had a [report about this in outlook before](https://github.com/code4recovery/tsml-ui/issues/234), but are unable to reproduce on other platforms.

**Reply by joshreisner** [reply]
here's something to try: could you fix the time in your outlook event and export it as a .ics file and attach it here? then we can compare it with the original, maybe there is a special setting outlook needs to see.

**Reply by Grateful-Viv** [reply]
I am not using Outlook and don't have it installed. I don't use Google calendar either. I maintain my events in iOS which works well with TSML. I need to check what the user has. 

When I tried it on my Windows desktop, I simply clicked the Add to Calendar option and it downloaded an ics file. That was the second image I uploaded.

 It won't let me attach it here.

**Reply by joshreisner** [reply]
hm! sounds like we need someone who has windows to do some research for us to help us id the issue. i'll ask in the code4recovery slack

**Reply by Grateful-Viv** [reply]
Weird. I did import the ics into Google calendar. It was the correct time. I'll reach out to the customer and see what he's using. I will then suggest he import it, fix the time, and then export it back to me. I'll rename the extension or just zip it.

![Google-Calendar](https://user-images.githubusercontent.com/62247421/230496551-2b970b83-e6cc-4568-bc8b-9a4f7bfd1c01.png)

**Reply by joshreisner** [reply]
ok thanks! yes google calendar import ics works for me too

when you get the file back you could also try opening it with a text editor and pasting the text in github.

**Reply by Grateful-Viv** [reply]
He is using Windows 10 Enterprise (v21H2) and Outlook for Microsoft 365 MSO (Version 2212 Build 16.0.15928.20196) 32-bit.

I did a bit of searching and this seems to be common with Outlook and Windows when DST happens. Here is one solution which suggests changing the start of the week to Monday instead of Sunday. After this week, they can switch it back to Sunday if they want. 
https://community.jamf.com/t5/jamf-pro/outlook-calendar-showing-1-hour-off/m-p/161433 

Here's the two ICS files for that meeting.

[Thursday 11th Step ICS.zip](https://github.com/code4recovery/12-step-meeting-list/files/11179180/Thursday.11th.Step.ICS.zip)

**Reply by joshreisner** [reply]
thanks! do you know if that fixed his issue?

**Reply by Grateful-Viv** [reply]
I don’t know. Haven’t heard back yet.

**Reply by Grateful-Viv** [reply]
Finally heard back. Changing the week start day had no effect. I’m inclined to believe it’s more an outlook issue than TSML.
